### PR TITLE
New: [AEA-4488] - Adds NPPTS Slack Alerts

### PR DESCRIPTION
**Pull request by JackSpagnoliNHS**
## Summary

- :sparkles: New Feature

### Details

Adds NPPTS Slack Alerts

### CODE REVIEW

File: SAMtemplates/alarms/main.yaml [hunk 1]
```diff
@@ -81,3 +88,175 @@ Resources:
...
+          Unit: Count
+          Dimensions:
+            - Key: FunctionName
+              Value: $.function_name
+
+  CPSUFhirConverterErrorsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmDescription: Count of CPSU FHIR Converter errors
+      AlarmName: !Sub ${StackName}_CPSU_FHIR_Converter_Errors
```
Comment (Orkastrated) [inline SAMtemplates/alarms/main.yaml:243]: So i think the extra underscores in the alarm name will end up with the alarm name coming through to slack being a bit wrong, may need to concat the middle section so it ends up something like ${StackName}_CpsuFhirConverter_Errors

Comment (JackSpagnoliNHS) [reply SAMtemplates/alarms/main.yaml]: 27a3173c67787437bd27d7e821cbf8b84014a017

File: SAMtemplates/state_machines/main.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ Parameters:
   EnableSplunk:
     Type: String
 
-  FihrValidatorStackName:
+  FhirValidatorStackName:
```
Comment (wildjames) [inline SAMtemplates/state_machines/main.yaml:33]: Awestruck that you spotted this

Comment (JackSpagnoliNHS) [reply SAMtemplates/state_machines/main.yaml:33]: I would take the credit but I only saw it when cloudformation kicked off when I tried to import it spelt correctly 😹

**Review by wildjames — APPROVED** [review]

**Pull request merged**